iosxr_setup.py

```diff
@@ -36,6 +36,25 @@ def open_sessions(self, username, password):
         self.xr_telnet_sessions.append(self.xr1)
 
     #
+    # Send a global-vrf XR Linux command - wait for prompt
+    # Could do bash -c or run ip netns exec global-vrf bash
+    #
+    def send_operns(self, command):
+        xr1 = self.xr1
+
+        xr1.send("bash -c %s" % command)
```

Comment:
@glennmatthews @goblinhack I think this is the only part that I haven't changed - xrnns and resolv.conf. I'll keep looking at it but let me know if other changes look ok?